### PR TITLE
feat: #1106  add align styles to labelsModal

### PR DESCRIPTION
**Pull request by AKiroi**
https://github.com/user-attachments/assets/10ae7be3-84d3-4128-ad9f-21c218664eb9

### CODE REVIEW

File: mw-webapp/src/logic/wayPage/WayPage.module.scss [hunk 1]
```diff
@@ -155,9 +155,15 @@ $iconContainerSize: 25px;
...
 }
 
-.jobDoneTagsWrapper {
-  min-width: $minWidthBlock;
-  max-width: $maxWidthBlock;
+.labelsWrapper {
+  min-width: 650px;
+
+  @media only screen and (max-width: $mediaWidthTablet) {
+    min-width: 350px;
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/WayPage.module.scss:162]: $mediumWidthBlock

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss [hunk 2]
```diff
@@ -24,27 +25,33 @@ $descriptionMaxWidthMobile:150px;
 
 .labelContent {
   align-items: center;
+  justify-content: space-between;
   gap: $gapMedium;
 }
 
 .labelContainer {
+  display: flex;
```
Comment (Ekaterina1994) [inline mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss:33]: maybe better to update div => HorizontalContainer instead property flex

File: mw-webapp/src/logic/wayPage/WayPage.module.scss (hunk 1 above)
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/WayPage.module.scss:162]: done

File: mw-webapp/src/logic/wayPage/WayPage.module.scss [hunk 3]
```diff
@@ -155,9 +155,15 @@ $iconContainerSize: 25px;
   gap: $gapExtraSmall;
 }
 
-.jobDoneTagsWrapper {
-  min-width: $minWidthBlock;
-  max-width: $maxWidthBlock;
+.labelsWrapper {
+  min-width: 650px;
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/WayPage.module.scss:159]: done

File: mw-webapp/src/logic/wayPage/labels/LabelLine.module.scss [hunk 4]
```diff
@@ -1,9 +1,12 @@
...
 
 .labelLine {
-  align-items: baseline;
-  gap: $gapExtraSmall $gapBig;
+  display: flex;
   flex-wrap: wrap;
+  align-items: center;
+  // justify-content: space-between;
+  // gap: $gapExtraSmall $gapBig;
+  gap: $gapMedium;
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/labels/LabelLine.module.scss:9]: done

File: mw-webapp/src/logic/wayPage/labels/LabelLine.module.scss [hunk 5]
```diff
@@ -1,9 +1,12 @@
 @import "src/styles/variables";
 
 .labelLine {
-  align-items: baseline;
-  gap: $gapExtraSmall $gapBig;
+  display: flex;
```
[the remark this answers is not in the document]
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/labels/LabelLine.module.scss:4]: done

File: mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss (hunk 2 above)
Comment (AKiroi) [reply mw-webapp/src/logic/wayPage/reportsTable/modalContentLabels/ModalContentLabels.module.scss:33]: done

**Review by Ekaterina1994 — APPROVED** [review]

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**